config-files/CYGNSS_L2_SURFACE_FLUX_V3.2.cfg

```diff
@@ -0,0 +1,85 @@
+{
+    "shortName": "CYGNSS_L2_SURFACE_FLUX_V3.2",
+    "latVar": "lat",
+    "lonVar": "lon",
+    "timeVar": "time",
+    "is360": true,
+    "tiles": {
+        "steps": [
+            30,
+            14
+        ]
+    },
+    "footprint": {
```

Comment:
@yiboj This shouldn't have footprint since it's a global bbox.  Can you update this?